app/config/FrontendAppConfig.scala

```diff
@@ -56,9 +56,15 @@ class FrontendAppConfig @Inject() (configuration: Configuration, servicesConfig:
   val enrolmentServiceName: String   = configuration.get[String]("enrolment.serviceName")
   val enrolmentIdentifierKey: String = configuration.get[String]("enrolment.identifierKey")
 
-  def getContactPreferenceUrl(appaId: String): String =
-    s"$contactPreferencesHost/alcohol-duty-contact-preferences/contact-preference/$appaId"
+  def ecpUserAnswersGetUrl(appaId: String): String =
+    s"$contactPreferencesHost/alcohol-duty-contact-preferences/user-answers/$appaId"
 
-  def setContactPreferenceUrl(appaId: String): String =
-    s"$contactPreferencesHost/alcohol-duty-contact-preferences/update-contact-preference/$appaId"
+  def ecpUserAnswersUrl(): String =
+    s"$contactPreferencesHost/alcohol-duty-contact-preferences/user-answers"
+
+  def ecpReleaseUserAnswersLockUrl(appaId: String): String =
```

Comment:
I was wondering if we actually need a lock for ECP as it's a 15 minute journey and it's unlikely multiple please will try and update the same email address at the same time.  I will have a think about this one. 🤔